hs/t/n/mint-fail2.txt

```diff
@@ -9,17 +9,17 @@ Verification failed:
 
   // Violation Witness
 
-  const UInt.max = 30880;
+  const UInt.max = 35424;
```

Comment:
I bet these numbers are different because you have a different Z3 than the docker image/CI

Reply:
yea, was just looking at this. I'll rollback the change and ignore